extensions/oidc-common/runtime/src/main/java/io/quarkus/oidc/common/runtime/OidcCommonConfig.java

```diff
@@ -425,97 +414,95 @@ public void setKey(String key) {
          * Default is required.
          */
         @ConfigItem
         public Optional<Verification> verification = Optional.empty();
 
         /**
-         * An optional keystore which holds the certificate information instead of specifying separate files.
+         * Optional. A keystore file that holds the certificate information instead of specifying separate files.
          */
         @ConfigItem
         public Optional<Path> keyStoreFile = Optional.empty();
 
         /**
-         * An optional parameter to specify type of the keystore file. If not given, the type is automatically detected
-         * based on the file name.
+         * Optional. The type of the keystore file. If not given, the type is automatically detected based on the file name.
          */
         @ConfigItem
         public Optional<String> keyStoreFileType = Optional.empty();
 
         /**
-         * An optional parameter to specify a provider of the keystore file. If not given, the provider is automatically
-         * detected
-         * based on the keystore file type.
+         * Optional. The provider of the keystore file. If not given, the provider is automatically detected based on the keystore file type.
          */
         @ConfigItem
         public Optional<String> keyStoreProvider;
 
         /**
-         * A parameter to specify the password of the keystore file. If not given, the default ("password") is used.
+         * Optional. The password of the keystore file. If not given, the default value, `password`, is used.
          */
         @ConfigItem
         public Optional<String> keyStorePassword;
 
         /**
-         * An optional parameter to select a specific key in the keystore. When SNI is disabled, if the keystore contains
-         * multiple
+         * Optional. The alias of a specific key in the keystore.
+         * When SNI is disabled, if the keystore contains multiple
          * keys and no alias is specified, the behavior is undefined.
          */
         @ConfigItem
         public Optional<String> keyStoreKeyAlias = Optional.empty();
 
         /**
-         * An optional parameter to define the password for the key, in case it's different from {@link #keyStorePassword}.
+         * Optional. The password of the key, if it is different from the {@link #keyStorePassword}.
          */
         @ConfigItem
         public Optional<String> keyStoreKeyPassword = Optional.empty();
 
         /**
-         * An optional truststore which holds the certificate information of the certificates to trust
+         * Optional. The truststore that holds the certificate information of the certificates to trust.
          */
         @ConfigItem
         public Optional<Path> trustStoreFile = Optional.empty();
 
         /**
-         * A parameter to specify the password of the truststore file.
+         * Optional. The password of the truststore file.
```

Comment:
@Sergey
As a new user reading these docs, does it help to say "* A parameter to specify", or is this redundant?